nucypher/policy/payment.py

```diff
@@ -10,7 +10,7 @@
     BaseContractRegistry,
     InMemoryContractRegistry,
 )
-from nucypher.policy.policies import Policy
+from nucypher.policy import policies
```

Comment:
Was there an issue that necessitated no longer importing `Policy` directly...seems like it was fine before this change...?

Reply:
This is from 3159 which has already been merged